README.md

```diff
@@ -1,18 +1,58 @@
 # Running Kubernetes Anywhere
 
-This project uses Weave to dramatically simplify Kubernetes deployment - anywhere. It is by far the easiest way to get started on a single machine, and later scale-out to any infrastructure seamlessly. We use Weave Net as a cluster management network. This enables complete portability, and for example allows one to move or clone the entire cluster. Even TLS setup is fully transparent, and in-place upgrades are very easy.
+> ## TL;DR
+>
+> Make Kubernetes cluster setup super easy with Weave Net. Pick your tools and your cloud,
+> but no need to set it all up from scratch. Several fully-automated provisioning examples
+> [are provided](#enough-said--go-try-it).
 
-Additionally, thanks to how [Weave Net handles IP address allocation as well as DNS](https://www.youtube.com/watch?v=117gWVShcGU) without requiring a persistent store, you can deploy etcd over Weave Net as well. The etcd cluster can thereby benefit from simple service discovery WeaveDNS provides and therefore facilitate node replacement without config changes.
+Kubernetes Anywhere uses Weave Net to dramatically simplify Kubernetes deployment — anywhere.
+It is by far the easiest way to get started on a single machine and to subsequently scale-out
+to any infrastructure. By implementing Weave Net as a cluster management/bootstrap network,
+complete portability is ensured, allowing you to move or clone an entire cluster. Even provisioning
+of security certificates is fully transparent, and, needless to say, in-place cluster upgrades
+are very easy, as everything is containerised.
 
-Now you can simply configure all of the cluster components to have fixed DNS names, all you should care about is how these services are distributed accross your compute instances, e.g. what is the size of etcd cluster and whether it is on a dedicated machines with the right type of storage attached.
+Because Weave Net manages its state without [the use of an external store][bryans-talk], you
+can also deploy etcd with the help of Weave Net. An etcd cluster also benefits from the service
+discovery that WeaveDNS provides, making etcd management easier.
 
-You no longer have to care about the IP address of the API server or any of those things.
+In the nutshell, each of the cluster components gets a fixed DNS names, e.g. `kube-apiserver.weave.local`,
+`etcd1.weave.local` ...etc. Having this enables you to focus on higher level problems, i.e. make
+cluster bootstrap the responsibility of the network and put less weight on already complex provisioning
+and management tools.
 
-# Try it!
+[bryans-talk]: https://www.youtube.com/watch?v=117gWVShcGU
+
+# What is it made of?
```

Comment:
Do we really need this section? Here? It tells me stuff w/o context. Why would I care that we ship discrete containers? Why would I want to build them myself? etc. All sounds like stuff for advanced users, hence should come later, and needs explaining.